continuous_integration/environment-3.8-dev.yaml

```
@@ -3,32 +3,32 @@ channels:
 - conda-forge
 - nodefaults
 dependencies:
-- dask-ml=2022.1.22
-- dask=2022.3.0
-- fastapi=0.69.0
-- fugue=0.7.0
-- intake=0.6.0
+- dask-ml>=2022.1.22
+- dask>=2022.3.0
+- fastapi>=0.69.0
+- fugue>=0.7.0
+- intake>=0.6.0
```

Could we switch these back to being pinned? This is to make sure our min versions are covered by testing:

```suggestion
- dask-ml=2022.1.22
- dask=2022.3.0
- fastapi=0.69.0
- fugue=0.7.0
- intake=0.6.0
```

---

Yep, I thought it was just an oversight since the 3.9 and 3.10 files are this way. Makes sense though.